babel.config.js

```diff
@@ -0,0 +1,59 @@
+module.exports = api => {
+  const isTest = api.env("test")
+  return {
+    env: {
+      test: {
+        plugins: ["dynamic-import-node"],
+      },
+    },
+    presets: [
+      "@babel/typescript",
+      "@babel/react",
+      [
+        "@babel/env",
+        {
+          useBuiltIns: "usage",
+          modules: isTest ? "commonjs" : false,
```

Comment:
The static version of this file should support switching between these two, but I wasn't sure about how the merge order would be handled, so I broke it out into this file format.